modules/appeals_api/spec/requests/v1/notice_of_disagreements_controller_spec.rb

```diff
@@ -26,6 +26,8 @@ def base_path(path)
     context 'creates an NOD and persists the data' do
       it 'with all headers' do
         post(path, params: @data, headers: @headers)
+        nod = AppealsApi::NoticeOfDisagreement.last
```

Comment:
Could you pull the id of the NOD from the response instead of inferring it here?

Reply:
Agreed, there's no guarantee that `.last` will actually return the most recent entry from the database, since without an ORDER clause the database makes no guarantees about order.

`parsed` should have the guid already 👍